03-Mains/Heaving_Water_mass_transformation_decomposition.ipynb

```diff
@@ -0,0 +1,2108 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #2.    experiment_Control = '01deg_jra55v13_ryf9091' # Control experiment
</pre><p>Suggest either snake_case or CamelHumping for variable names. The combination is a bit odd. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/COSIMA/cosima-recipes/pull/620/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>